KPI/pyproject.toml

```diff
@@ -0,0 +1,23 @@
+[project]
+name = "baf-kpi"
+version = "0.0.1"
```

Comment:
I would add a comment here explaining that it should be aligned with the version number in https://github.com/ami-iit/biomechanical-analysis-framework/blob/main/CMakeLists.txt#L10 .